lib/src/views/post/edit_post_screen.dart

```diff
@@ -53,14 +55,16 @@ class _EditPostScreenState extends State<EditPostScreen> {
   bool isMediaPost = false;
   String previewLink = '';
   String convertedPostText = '';
-  MediaModel? linkModel;
+  AttachmentPostViewData? linkModel;
   List<UserTag> userTags = [];
   bool showLinkPreview =
       true; // if set to false link preview should not be displayed
   Timer? _debounce;
   Size? screenSize;
   ThemeData? theme;
   Map<String, WidgetModel>? widgets;
+  bool showRepost = false;
```

Comment:
remove this var if not needed

Reply:
we are checking for attachment type == 8 and then storing this value to true to display repost widget